tests/test_oci.py

```diff
@@ -512,6 +515,38 @@ async def mock_run_async_subprocess(
         prov = await client.fetch_latest_provenance(image)
         assert prov.predicate == make_provenance_predicate(new_date)
 
+    @pytest.mark.asyncio
+    async def test_fetch_attested_sbom(
+        self,
+        image: Image,
+        client: CosignClient,
+        monkeypatch: pytest.MonkeyPatch,
+        sbom_doc: dict[str, Any],
+    ) -> None:
+        expected_payload = {
+            "predicateType": "https://in-toto.io/Statement/v0.1",
+            "predicate": sbom_doc,
+        }
+        expected_attestation = {
+            "payload": base64.b64encode(json.dumps(expected_payload).encode()).decode()
+        }
+        attestation_raw = json.dumps(expected_attestation).encode()
+
+        async def mock_perform_cosign_verify_attestation(*_: Any) -> list[bytes]:
+            return [attestation_raw]
```

Comment:
Nitpick: This is not a mock, but a stub.
https://www.martinfowler.com/articles/mocksArentStubs.html#TheDifferenceBetweenMocksAndStubs

```suggestion
        async def fake_perform_cosign_verify_attestation(*_: Any) -> list[bytes]:
            return [attestation_raw]
```